webapp-django/crashstats/profile/views.py

```diff
@@ -0,0 +1,100 @@
+import datetime
+
+from django import http
+from django.contrib.auth.decorators import login_required
+from django.contrib.sites.models import RequestSite
+from django.shortcuts import render, redirect
+from django.db import transaction
+from django.contrib.auth.models import Permission
+
+from crashstats.crashstats.decorators import pass_default_context
+from crashstats.supersearch.models import SuperSearchUnredacted
+from crashstats.tokens import models
+from crashstats.tokens import forms
+
+
+@pass_default_context
+@login_required
+@transaction.atomic
```

Comment:
Oh. I was wrong. This view DOES create data. 
However, I think it's a mistake to repeat the code for creating tokens here. It should be done over in the API Tokens view.